ui/packages/catalog-ui-search/src/main/webapp/component/result-form/result-form.js

```diff
@@ -46,7 +46,7 @@
 
                   customResultTemplates.push({
                     label: 'All Fields',
-                    value: 'All Fields',
+                    value: 'allFields',
```

Comment:
Value may need to be 'All Fields'.  When this change was in place on 1.4 it would place the lowercase value in the QuerySettings dropdown.

Reply:
I believe the `label` should be what is displayed. It falls back to the value if a label isn't defined, which in our case is what we assign the [title](https://github.com/codice/ddf/pull/3178/files#diff-638762e92207d31822cc29a9020420d2R39) attribute of the form to